notion_client/api_endpoints.py

```diff
@@ -325,3 +325,29 @@ def list(self, **kwargs: Any) -> SyncAsync[Any]:
             query=pick(kwargs, "block_id", "start_cursor", "page_size"),
             auth=kwargs.get("auth"),
         )
+
+
+class OAuthEndpoint(Endpoint):
+    def introspect(self, **kwargs: Any) -> SyncAsync[Any]:
+        """Introspect the provided token.
+
+        *[🔗 Endpoint documentation](https://developers.notion.com/reference/introspect-token)*
+        """
```

Comment:
```suggestion
        """  # noqa: E501
```

Reply:
Hmm, curious that the CI doesn't break without these. Maybe we can remove them all for some reason?

Reply:
I can add them to be consistent with the rest of the SDK if that would be best.